src/plugins/ganttChart/ganttChart.js

```diff
@@ -203,6 +202,7 @@ class GanttChart extends BasePlugin {
       return;
     }
 
+    warn('Gantt Chart plugin is deprecated and will be removed');
```

Comment:
Might be nice to add some more context
```suggestion
    warn('Gantt Chart plugin is deprecated and will be removed in the next major release');
```